test/sanity/pep8/legacy-files.txt

```diff
@@ -1,3 +1,8 @@
+hacking/conf2yaml.py
+lib/ansible/constants.py
+lib/ansible/cli/config.py
+lib/ansible/config/data.py
+lib/ansible/config/manager.py
```

Comment:
most of the code was moved from constants,py which was under it (someone refactored and removed it while this PR was in flight, so that is why it looks like i re added constants.py)